pkg/brokerapi/fake/server/update_request.go

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
It's mostly personal preference.

n.b. there are some best practices in Go that support this decision, but there are also conflicting ones too